docs/frameworkanforderungen.md

```diff
@@ -4,33 +4,33 @@
 
 ## Technisch
 
-ID | Anforderung | Gewichtung
----| ----------- | -------------
-1 | Apps für Android erstellen | muss
-2 | Apps für iOS erstellen | muss
-3 | Apps für Web-Browser erstellen | soll
-4 | Externe REST API anbinden | muss
-5 | Mehrere Bilder von Galerie beziehen | muss
-6 | Bild von Kamera beziehen | soll
+ID | Anforderung | Gewichtung | Quelle
+---| ----------- | ---------- | ------
+1 | Apps für Android erstellen | muss | FA13, FA14
+2 | Apps für iOS erstellen | muss | FA11, FA12
+3 | Apps für Web-Browser erstellen | soll | FA15
+4 | Externe REST API anbinden | muss | FA17
+5 | Mehrere Bilder von Galerie beziehen | muss | FA1
+6 | Bild von Kamera beziehen | soll | FA2
 7 | Unterstützt Unit-Testing | soll
```

Comment:
Should we have a requirements or non-functional requirement for Testing, CD. Is it maybe a customer wish to deploy fixes fast